Misc/NEWS.d/next/Build/2017-11-03-21-54-32.bpo-31940.fG1wnQ.rst

```diff
@@ -0,0 +1,2 @@
+Detect faulty ``lchmod`` implementation to fix ``shutil.copystat`` on alpine
```

Comment:
It might be useful to say "on Alpine linux and other distributions using musl libc", because the issue is actually related to the musl implementation, I believe. This affects me on Gentoo+musl.

Reply:
yep, I did hit this on gentoo musl as well

Reply:
oh shoot, I totally forgot about this patch -- I need to revive my other approach 😨